packages/config/config/devices/0x001d/dz6hd.json

```diff
@@ -50,30 +62,78 @@
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Memory Dim",
+					"value": 0
+				}
+			]
 		},
 		"6": {
 			"label": "LED Dim Level Indicator Timeout",
```

Comment:
I think this is missing the unit and the "Always on" (255) option.

Reply:
I missed it, adding now…